.prettierrc.mjs

```diff
@@ -0,0 +1,13 @@
+// .prettierrc.mjs
+/** @type {import("prettier").Config} */
+export default {
+  plugins: ['prettier-plugin-astro'],
+  overrides: [

```

Comment:
Perhaps I don't need this if I'm going to specify the formatter in my `languages.toml` file.

I'll take a look at this in another PR.